x-pack/plugins/inference/server/util/validate_tool_calls.test.ts

```diff
@@ -5,8 +5,8 @@
  * 2.0.
  */
 
-import { isToolValidationError } from '../../common/chat_complete/errors';
-import { ToolChoiceType } from '../../common/chat_complete/tools';
+import { ToolChoiceType } from '@kbn/inference-common';
+import { isToolValidationError } from '@kbn/inference-common/src/chat_complete/errors';
```

Comment:
nit: should we be reaching this far into the `inference-common` package for the import?

Reply:
https://github.com/elastic/kibana/pull/193464/commits/7904a14c8758c3454295ad04daf574c28bcfb605